### PR TITLE
Add pyPRISM

### PR DESCRIPTION
**Pull request by martintb**
Add pyPRISM to conda-forge.

### CODE REVIEW

File: recipes/pyprism/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy >=1.8.0
+    - scipy
+    - pint
```
Comment (CJ-Wright) [inline recipes/pyprism/meta.yaml]: is pint abuild requirement?

File: recipes/pyprism/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  run:
+    - python
```
Comment (CJ-Wright) [inline recipes/pyprism/meta.yaml]: Is numpy a run time requirement?

File: recipes/pyprism/meta.yaml (hunk 1 above)
Comment (martintb) [reply recipes/pyprism/meta.yaml]: Nope. Fixed.

File: recipes/pyprism/meta.yaml (hunk 2 above)
Comment (martintb) [reply recipes/pyprism/meta.yaml]: Yes. Added. Thanks!

**Comment by martintb** [comment]
@CJ-Write
@conda-forge/staged-recipes

As per the README, pinging after a week. All tests are passed. Are there any other changes I should make to the package? Thanks!

**Comment by CJ-Wright** [comment]
Thank you for the ping!

**Comment by CJ-Wright** [comment]
LGTM

File: recipes/pyprism/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    - numpy
+    - scipy
+    - pint
+
+test: 
+  requires:
+    - pytest
+    - numpy
```
Comment (CJ-Wright) [inline recipes/pyprism/meta.yaml]: This should be installed at runtime (so you don't need to include the run requirements).

File: recipes/pyprism/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+test: 
+  requires:
+    - pytest
+    - numpy 
+    - scipy
+    - pint
+  imports:
+    - pyPRISM
+    - numpy
```
Comment (CJ-Wright) [inline recipes/pyprism/meta.yaml]: Usually one just imports the package modules here (rather than also importing the deps).

Comment (martintb) [reply recipes/pyprism/meta.yaml]: So, just change this to only include pyPRISM?

Comment (martintb) [reply recipes/pyprism/meta.yaml]: Is this correct? 
```
test:
    requires:
        - pytest
    imports:
        - pyPRISM
```

Comment (CJ-Wright) [reply recipes/pyprism/meta.yaml]: I think so. you can also include submodules if you want.

File: recipes/pyprism/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    - scipy
+    - pint
+
+test: 
+  requires:
+    - pytest
+  imports:
+    - pyPRISM
+    - pyPRISM.core
```
Comment (CJ-Wright) [inline recipes/pyprism/meta.yaml]: :+1:

**Comment by CJ-Wright** [comment]
LGTM, thank you very much!

**Comment by CJ-Wright** [comment]
Next time please PR from a branch on your fork (rather than master).

**Comment by CJ-Wright** [comment]
Welcome to conda-forge!

**Pull request merged**